openc3-cosmos-init/plugins/packages/openc3-tool-common/src/services/cable.js

```diff
@@ -44,4 +44,7 @@ export default class Cable {
       )
     })
   }
+  recordPing() {
+    this._cable.connection.monitor.recordPing()
+  }
```

Comment:
Did this have something to do with another issue? I remember a conversation about ping and the connection timeout.